packages/boot/src/booters/datasource.booter.ts

```diff
@@ -47,9 +47,9 @@ export class DataSourceBooter extends BaseArtifactBooter {
     await super.load();
 
     /**
-     * If Repository Classes were discovered, we need to make sure RepositoryMixin
+     * If DataSource Classes were discovered, we need to make sure RepositoryMixin
      * was used (so we have `app.repository()`) to perform the binding of a
```

Comment:
Actually, I think this should say `app.dataSource()`, not `app.repository()`!

Reply:
@bajtos Great catch! Fixed.